### ISSUE TITLE
provide contact formular

### ISSUE THREAD

**Issue by justus-reihs** [lnnrtwttkhn/version-control-book#253]
To contribute to the vcd, one has to use Git and GitHub. The aim of the book is to teach people git for version control. I would prefer a contact formular for people that are just starting.

**Comment by lnnrtwttkhn** [comment]
Good idea, thanks @justus-reihs! I think I would like to try "Hypothesis" (see [here](https://quarto.org/docs/reference/projects/websites.html#hypothesis))
